wpeverest/user-registration
# Enhance - Option to disable logout confirmation

### All Submissions:






### Changes proposed in this Pull Request:
This PR introduces an option to disable logout confirmation.

### How to test the changes in this Pull Request:

1. Go to **Settings > General > My account Section**
2. Check the **Disable logout confirmation** option
3. Now go to my account page and click on the Logout endpoint. You should be directly logged out.
4. Now uncheck the option
5. Try to logout again. Logout confirmation message should show up.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Option to disable logout confirmation
